index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio Website</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"
+    />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <ul class="menu-bar">
+          <li id="menu-icon">Welcome</li>
+          <li>
+            <div class="menu-icon1"><i class="fas fa-bars"></i></div>
+          </li>
+        </ul>
```

Comment:
Kindly remove the `ul` and `li` from the header and change the header to contain two `a` tags which are the logo ("Welcome" in your case) and the hamburger menu icon.